mvc-login/build.gradle

```
@@ -6,23 +6,22 @@ buildscript {
 
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.7.0'
+    id 'org.springframework.boot' version '3.1.4'
     id 'io.spring.dependency-management' version '1.1.0'
```

```suggestion
    id 'io.spring.dependency-management' version '1.1.3'
```

This is the latest version if you create a new app with start.spring.io.

---

👍 incorporated in commit along with other changes